release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -174,6 +174,20 @@ The administrator may review the referenced materials and choose to perform the
 
 For more information, see xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Conditional updates] and xref:../updating/updating-cluster-cli.adoc#update-conditional-upgrade-pathupdating-cluster-cli[Updating along a conditional upgrade path].
 
+[id="ocp-4-10-installing-cluster-on-openstack-ovs-dpdk"]
+==== Installing a cluster on {rh-openstack} that uses OVS-DPDK
+
+You can now install a cluster on {rh-openstack-first} for which compute machines run on Open vSwitch with the Data Plane Development Kit (OVS-DPDK) networks and have their workloads benefit from it. 
```

Comment:
I'm not sure what you mean by "have their workloads benefit from it." Will you please take another look and consider replacing a pronoun or two?

Reply:
```suggestion
You can now install a cluster on {rh-openstack-first} for which compute machines run on Open vSwitch with the Data Plane Development Kit (OVS-DPDK) networks. Workloads that run on these machines can benefit from the performance and latency improvements of OVS-DPDK. 
```

Reply:
@EmilienM Are you okay with this proposed phrasing?